index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="main"> 
+        <div class="header">
+            <header>
+                <nav>
+                    <div>
+                        <a>
+                            <img class="logo" src="./image/logo.png" alt="">
+                        </a>
+                    </div>
+                    <div class="navbar">
+                        <div></div>
+                        <div></div>
+                        <div></div>
+                    </div>
+                </nav>
+            </header>
+        </div>
+        <div class="eclipse-3">
+            <img src="./image/Ellipse3(1).png" alt="">
+        </div>
+        <div class="body">
+            <div class="name">
+                I’m Saikat Glad to see you!
+            </div>
+            <div class="description">
```

Comment:
- [ ] As the sentence `I'M Saikat Gald to see you` contains a main title and a subtitle, It would be better if you use the `<h1>` tag for the main title and an `hx` tag for the subtitle `eg: h2, h3, h4...`. Please, Check the suggestion below
```suggestion
        <div class="body">
            <div class="name">
                <h1>I’m Saikat</h1>
                <h2> Glad to see you!</h2>
            </div>
            <div class="description">
```

Reply:
Changed the text style using h1 and h2 tag and reduced the font size 40 to 25px to stop the overflow.